libs/common/src/key-management/master-password/types/master-password.types.ts

```diff
@@ -0,0 +1,31 @@
+import { Opaque } from "type-fest";
+
+// eslint-disable-next-line no-restricted-imports
+import { KdfConfig } from "@bitwarden/key-management";
+
+import { EncString } from "../../crypto/models/enc-string";
+
+/**
+ * The Base64-encoded master password authentication hash, that is sent to the server for authentication.
+ */
+export type MasterPasswordAuthenticationHash = Opaque<string, "MasterPasswordAuthenticationHash">;
+export type MasterPasswordSalt = Opaque<string, "MasterPasswordSalt">;
+export type MasterKeyWrappedUserKey = Opaque<EncString, "MasterPasswordSalt">;
+
+/**
+ * The data required to unlock the master password.
```

Comment:
```suggestion
 * The data required to unlock with the master password.
```

Or maybe `via the master password.`? 

How it is now reads to me like `master password` is the subject of unlocking.